### ISSUE TITLE
Remove Bootstrap  - 508

### ISSUE THREAD

**Issue by JorjaComer** [US-EPA-CAMD/PI-1#1504]
## Required Fix 

- [x] RF-2 

## Context
Remove Bootstrap and update styling as needed  

See [EPA CAMD Accessibility Report ](https://teams.microsoft.com/l/file/31E8B6EF-1313-4495-B867-46347BB7E4A2?tenantId=88b378b3-6748-4867-acf9-76aacbeca6a7&fileType=docx&objectUrl=https%3A%2F%2Fusepa.sharepoint.com%2Fsites%2FCAMDCVPTeam%2FShared%20Documents%2FDatateers%2F5-UI%20Design%2F02%20508%20and%20Accessibility%2FEPA.CAMPD.Accessibility.Report%20.docx&baseUrl=https%3A%2F%2Fusepa.sharepoint.com%2Fsites%2FCAMDCVPTeam&serviceName=teams&threadId=19:f4349e2ba95e43f3b29e8208bc8f5d5f@thread.skype&groupId=21a5e1cd-d3f1-48a6-801f-0d34e2d63d23) for how to fix and additional details 

508 issue: Color contrast ratio between text and background is less than 4.5:1. This is a violation of HHS Guidelines and WCAG 2.0 Success Criterion 1.4.3. |

## Testing
Tester: @aprematta 

### Definition of Done
- [ ] Have all Acceptance Criteria been agreed to and validated by the scrum team?
- [ ] Has the code been unit tested and peer reviewed?
- [ ] Have the functional tests been executed?
- [ ] Have all defects been reviewed and dispositioned, resolved, or deferred?
- [ ] Has the user story been accepted by the Product Owner?
- [ ] Is the user story ready to be deployed to the test/staging environments?

**Comment by junie-stigdon** [comment]
## Pull Request

- removed bootstrap css stylesheet
- fixed styling for:
  - wide header
  - sub header
  - footer
  - select data homepage
    - headers
    - body text
    - selectable cards ** **(RF-2)** **
  - data type/subtype select
  - filter panel
  - filter criteria:
    - checkbox filters
    - multiselect combo box filters
  - filter tags
  - data preview header
  - download file selector

**Comment by aprematta** [comment]
Color contrast now meets 508, closing issue
